src/components/HeaderWithBackButton/index.tsx

```diff
@@ -178,9 +179,10 @@ function HeaderWithBackButton({
                 {!!icon && (
                     <Icon
                         src={icon}
-                        width={variables.iconHeader}
-                        height={variables.iconHeader}
-                        additionalStyles={[styles.mr2]}
+                        width={iconWidth ?? variables.iconHeader}
+                        height={iconHeight ?? variables.iconHeader}
+                        additionalStyles={[styles.mr2, iconStyles]}
+                        fill={iconFill ?? theme.icon}
```

Comment:
The `?? theme.icon` default caused https://github.com/Expensify/App/issues/54514